spec/ruby_basics_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+require_relative '../lib/ruby_basics.rb'
+
+
+describe "ruby" do
+
+  describe "#division" do
+    it 'given two numbers returns quotient of the first number divided by the second number' do
+      num1 = 42
+      num2 = 7
+
+      expect(division(num1,num2)).to eq(6)
+    end
+  end
+
+  describe '#assign_variable' do
+    it 'takes an argument of a persons name and assign it to a variable name' do
+      value = "Bob"
+
+      expect(assign_variable(value)).to eq(value)
```

Comment:
this is not really testing assignment. it's testing the return value. You could pass with `def assign_variable(x); x; end`